### PR TITLE
Bulk validation fixes ms.prod

### PR DESCRIPTION
**Pull request by rflynn27**

### CODE REVIEW

**Comment by rflynn27** [comment]
Global effort to fix build validation errors

@Lauragra 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: api-reference/beta/resources/automaticrepliesmailtips.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "automaticRepliesMailTips resource type"
 description: "MailTips about any automatic replies that have been set up on a mailbox."
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: outlook
```
Comment (Lauragra) [inline api-reference/beta/resources/automaticrepliesmailtips.md:6]: Please change to "mail".

File: api-reference/beta/resources/automaticrepliessetting.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "automaticRepliesSetting resource type"
 description: "Configuration settings to automatically notify the sender of an incoming email with a message from the "
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: outlook
```
Comment (Lauragra) [inline api-reference/beta/resources/automaticrepliessetting.md:6]: mail

File: api-reference/beta/resources/clonableteamparts.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "clonableTeamParts enum type"
 description: "Describes which part of a team should be cloned. "
 ms.localizationpriority: medium
 doc_type: enumPageType
-ms.prod: ""
+ms.prod: microsoft-teams
```
Comment (Lauragra) [inline api-reference/beta/resources/clonableteamparts.md:6]: Change to "teamwork".

File: api-reference/beta/resources/booleanColumn.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/11/2017
 title: BooleanColumn
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ""
+ms.prod: files
```
Comment (Lauragra) [inline api-reference/beta/resources/booleanColumn.md:8]: Please change to sites-and-lists.

File: api-reference/beta/resources/choicecolumn.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/11/2017
 title: ChoiceColumn
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: files
```
Comment (Lauragra) [inline api-reference/beta/resources/choicecolumn.md:8]: sites-and-lists

File: api-reference/beta/resources/columnLink.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/12/2017
 title: ColumnLink
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: files
```
Comment (Lauragra) [inline api-reference/beta/resources/columnLink.md:8]: sites-and-lists

File: api-reference/beta/resources/contentTypeOrder.md [hunk 7]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/13/2017
 title: ContentTypeOrder
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: files
```
Comment (Lauragra) [inline api-reference/beta/resources/contentTypeOrder.md:8]: sites-and-lists

File: api-reference/beta/resources/contenttypeinfo.md [hunk 8]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/12/2017
 title: ContentTypeInfo
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: sharepoint
```
Comment (Lauragra) [inline api-reference/beta/resources/contenttypeinfo.md:8]: sites-and-lists

File: api-reference/beta/resources/currencycolumn.md [hunk 9]
```diff
@@ -5,7 +5,7 @@ ms.date: 09/11/2017
 title: CurrencyColumn
 ms.localizationpriority: medium
 doc_type: resourcePageType
-ms.prod: ''
+ms.prod: files
```
Comment (Lauragra) [inline api-reference/beta/resources/currencycolumn.md:8]: sites-and-lists

**Review by Lauragra — APPROVED** [review]

**Comment by v-emilypr** [comment]
Hi @Lauragra, thank you for reviewing and approving this PR! Are you able to merge this also, we do not have the permissions to do so. Thank you!

**Pull request merged**